lib/spree_print_invoice/prawn_rails_configuration.rb

```diff
@@ -0,0 +1,32 @@
+module Spree
+  module PrintInvoice
+    class PrawnRailsConfiguration
+      def page_layout
+        Spree::PrintInvoice::Config.page_layout.to_sym
+      end
+
+      def page_size
+        Spree::PrintInvoice::Config.page_size
+      end
+
+      def skip_page_creation
+        false
+      end
+
+      def to_h
+        {
+          page_layout: Spree::PrintInvoice::Config.page_layout,
+          page_size: Spree::PrintInvoice::Config.page_size,
+          skip_page_creation: false
+        }
+      end
+    end
+  end
+end
+
+# A bit of a hack, but we really do need to modify PrawnRails' config all the time.
+module PrawnRails
+  def config
+    Spree::PrintInvoice::PrawnRailsConfiguration.new
```

Comment:
I guess you could cache this in an instance variable. Because instance variables get instantiated on each request, but if you use the `||=` it won't get instantiated more then once on each request. That would spare some database calls from the views I presume.